projects/packages/post-list/tests/php/test-post-list.php

```diff
@@ -104,8 +104,8 @@ public function test_add_filters_and_actions_for_screen_thumbnail() {
 		$this->assertTrue( has_action( 'manage_posts_custom_column' ) );
 		$this->assertTrue( has_filter( 'manage_pages_columns' ) );
 		$this->assertTrue( has_action( 'manage_pages_custom_column' ) );
-		$this->assertFalse( has_action( 'post_row_actions' ) );
-		$this->assertFalse( has_action( 'page_row_actions' ) );
+		$this->assertFalse( has_action( 'post_row_actions', 'add_share_action' ) );
+		$this->assertFalse( has_action( 'page_row_actions', 'add_share_action' ) );
```

Comment:
I also wanted to add checks like: 
```
$this->assertTrue( has_action( 'post_row_actions', 'add_copy_link_action' ) );
```

but they were failing for some reason. `global $wp_filter;` had `post_row_actions` inside but the returned value was `false`.

Any ideas why this is happening?

Reply:
Figured this out. We had to pass the instance too of the function called.